Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx

```diff
@@ -140,16 +140,16 @@ const AssetChannelWindow = (props: IProps) => {
         </div>
 
     return (
-        <div className="card" style={{ marginBottom: 10 }}>
+        <div className="card" style={{ marginBottom: 10, display: 'flex', flexDirection: 'column', maxHeight: '70vh' }}>
             <div className="card-header">
                 <div className="row">
                     <div className="col">
                         <h4>Channels:</h4>
                     </div>
                 </div>
             </div>
-            <div className="card-body">
-                <div style={{ width: '100%', maxHeight: window.innerHeight - 381, padding: 30 }}>
+            <div className="card-body" style={{ flex: 1, display: 'flex', flexDirection: 'column', overflowY: 'auto' }}>
+                <div style={{ flex: 1 }} >
```

Comment:
This div is unnecesarry. the Table should take up `100%` of the parent (`card-body`) already